docker/test_env/eth2_common.go

```diff
@@ -2,10 +2,14 @@ package test_env
 
 import (
 	"context"
+	_ "embed"
 	"fmt"
 	"testing"
 	"time"
 
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/pelletier/go-toml/v2"
+	"github.com/pkg/errors"
```

Comment:
Another usage of a no longer maintained library. Preferable to use fmt.Errorf for now.